src/test/cli/rbd/help.t

```diff
@@ -26,7 +26,7 @@
       config pool set                   Set a pool-level configuration override.
       copy (cp)                         Copy src image to dest.
       create                            Create an empty image.
-      deep copy (deep cp)               Deep copy src image to dest.
+      deep copy (deep cp)               Deep copy (including snapshots) src image to dest.
```

Comment:
Need to fix the expected output:

```
@@ -26,7 +26,8 @@
       config pool set                   Set a pool-level configuration override.
       copy (cp)                         Copy src image to dest.
       create                            Create an empty image.
-      deep copy (deep cp)               Deep copy (including snapshots) src image to dest.
+      deep copy (deep cp)               Deep copy (including snapshots) src image
+                                        to dest.
       device attach                     Attach image to device.
       device detach                     Detach image from device.
       device list (showmapped)          List mapped rbd images.
```

Reply:
Yep.

Having output of --help included in tests, and with different formatting, is a bit strange.

Reply:
`rbd help` output is generated on the fly and the test is there to make sure it doesn't go sideways.